src/styles/components/community/_contributors.scss

```
@@ -159,6 +159,10 @@
     font-weight: $bold;
   }
 
+  .contributor__badge {
+    width:40px;
```

```suggestion
    width: 40px;
```

We should add a CS fixer :stuck_out_tongue: